tools/mo/openvino/tools/mo/back/offline_transformations.py

```diff
@@ -36,6 +36,8 @@ def apply_user_transformations(func: object, transforms: list):
 
 def apply_moc_transformations(func: object):
     from openvino._offline_transformations import apply_moc_transformations  # pylint: disable=import-error,no-name-in-module
+    from openvino._offline_transformations import apply_smart_reshape_transformation  # pylint: disable=import-error,no-name-in-module
+    apply_smart_reshape_transformation(func)
     apply_moc_transformations(func, False)
```

Comment:
Why can't we move this transformation inside `moc_transformation` and perform it first? Now I see no reason to get this outside since this transformation is not applicable or useless without `moc_transformation`.

Reply:
I see that there is information in the ticket that we need enable it before moc pipeline. @mateusztabaka Is it still relevant?

Reply:
Ok, but can be apply_moc_transformations used without apply_smart_reshape_transformation?